packages/compass-query-bar/src/components/query-bar.tsx

```diff
@@ -78,65 +80,52 @@ const queryOptionsContainerStyles = css({
 const queryBarDocumentationLink =
   'https://docs.mongodb.com/compass/current/query/filter/';
 
+const QueryMoreOptionsToggle = connect(
+  (state: QueryBarState) => {
+    return { isExpanded: state.expanded };
+  },
+  { onToggleOptions: toggleQueryOptions }
+)(MoreOptionsToggle);
+
 type QueryBarProps = {
   buttonLabel?: string;
-  expanded: boolean;
-  globalAppRegistry: AppRegistry;
-  localAppRegistry: AppRegistry;
   onApply: () => void;
-  onChangeQueryOption: (queryOption: QueryOption, value: string) => void;
-  onOpenExportToLanguage: () => void;
   onReset: () => void;
-  queryOptionsLayout?: (QueryOption | QueryOption[])[];
+  onOpenExportToLanguage: () => void;
+  layout?: (QueryOption | QueryOption[])[];
   queryState: 'apply' | 'reset';
-  refreshEditorAction: Listenable;
-  resultId: string | number;
-  schemaFields: CompletionWithServerInfo[];
-  serverVersion: string;
+  resultId?: string | number;
   showExportToLanguageButton?: boolean;
   showQueryHistoryButton?: boolean;
-  toggleExpandQueryOptions: () => void;
   valid: boolean;
-} & QueryBarOptionProps;
+  expanded: boolean;
+  placeholders?: Record<QueryProperty, string>;
+};
 
-const QueryBar: React.FunctionComponent<QueryBarProps> = ({
+export const QueryBar: React.FunctionComponent<QueryBarProps> = ({
   buttonLabel = 'Apply',
-  expanded: isQueryOptionsExpanded = false,
-  globalAppRegistry,
-  localAppRegistry,
-  onApply: _onApply,
-  onChangeQueryOption,
-  onOpenExportToLanguage,
+  onApply,
   onReset,
+  onOpenExportToLanguage,
   // Used to specify which query options to show and where they are positioned.
-  queryOptionsLayout = [
```

Comment:
In the plugin `propTypes` this prop was called `queryOptions`, in mms they seem to use `layout`, and the actual expected prop was `queryOptionsLayout`, as there was no consistency, I changed to what mms uses

Reply:
Just saw that they recently updated in mms to `queryOptionsLayout`, will change this back

Reply:
Done in 9d732f2bc1689c8ad0aa9993f8d31264fb7af0c9